process/sync/baseSync.go

```diff
@@ -1005,6 +1019,10 @@ func (boot *baseBootstrap) GetNodeState() core.NodeState {
 	if boot.isInImportMode {
 		return core.NsNotSynchronized
 	}
+	currentSyncedEpoch := boot.getEpochForCurrentBlock()
```

Comment:
Should be tested to see if it works as expected at the edge of epoch change for the synced nodes

Reply:
As seen here https://github.com/ElrondNetwork/elrond-go/blob/optimize-full-archive-sync/dataRetriever/factory/epochProviders/currentEpochProvidersFactory.go we are constructing a disabled statusEpochProvider in case we are not in full archive mode. The disabledEpochProvider always returns true if queried for the EpochIsActiveInNetwork https://github.com/ElrondNetwork/elrond-go/blob/optimize-full-archive-sync/dataRetriever/resolvers/epochproviders/disabled/disabledEpochProvider.go

Reply:
ok